src/server/middlewares/redirects.ts

```diff
@@ -32,11 +32,14 @@ export const startPageRedirect: IMiddleware<object> = async (ctx) => {
     lookupCountry(actualIp)?.toLowerCase() || fallbackCountryLabel
   const queryStringMaybe = ctx.querystring ? '?' + ctx.querystring : ''
 
+  ctx.status = 301
+
   if (Object.keys(locales).includes(countryLabel)) {
-    return ctx.redirect(`/${countryLabel}${queryStringMaybe}`)
+    ctx.set('Location', `/${countryLabel}${queryStringMaybe}`)
+    return
   }
 
-  ctx.redirect(`/${fallbackCountryLabel}${queryStringMaybe}`)
+  ctx.set('Location', `/${fallbackCountryLabel}${queryStringMaybe}`)
```

Comment:
How is `ctx.set` different from `ctx.redirect`? I thought adding `ctx.status = 301` would be sufficient? @palmenhq

Reply:
i thought so also but it seems `ctx.redirect` overwrites with a `302`. I guess we could use `ctx.status` after `ctx.redirect` but i'm quite confident they will have the exact same effect or so i'm assuming, since setting the `Location` header + `301`/`302`(/`307`) is how you perform redirects with HTTP

Reply:
![image](https://user-images.githubusercontent.com/42962286/99062186-df869180-25a2-11eb-95c1-b3ef67783de7.png)
☝️ From the koa docs though.

Reply:
Hm strange, i must've been to eager while testing or something because i'm soo sure i got a 302. But maybe the server hadn't restarted yet or something. I can change it back and see what happens

Reply:
Yup, it worked1

Reply:
Cool! 🆒 🤠